sharded_queue/api.lua

```diff
@@ -319,24 +292,30 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
+        {{ tube_name = tube_name }}, {uri_list=storages})
 
-    if not ok or stats_collection == nil then
-        log.info(stats_collection)
-        return
+    if err then
+        error(err)
     end
+
+    if next(stats_collection) == nil then
+        return nil
+    end
+
     -- merge
-    local stat = stats_collection[1]
-    for i = 2, #stats_collection do
-        for name, count in pairs(stats_collection[i].tasks) do
-            stat.tasks[name] = stat.tasks[name] + count
+    local stat = { tasks = {}, calls = {} }
+
+    for _, uri_stat in pairs(stats_collection) do
+        for name, count in pairs(uri_stat.tasks) do
+            stat.tasks[name] = (stat.tasks[name] or 0) + count
         end
-        for name, count in pairs(stats_collection[i].calls) do
-            stat.calls[name] = stat.calls[name] + count
+        for name, count in pairs(uri_stat.calls) do
+            stat.calls[name] = (stat.calls[name] or 0) + count
         end
     end
     --
+
```

Comment:
Two empty lines are redundant. Please, drop this change.

Reply:
Done https://github.com/tarantool/sharded-queue/pull/19/commits/4be5d204dc0fa1f70f7c10160f3096402e9f6d3a